app/tasks/maintenance_tasks/task.rb

```diff
@@ -3,6 +3,8 @@ module MaintenanceTasks
   # Base class that is inherited by the host application's task classes.
   class Task
     extend ActiveSupport::DescendantsTracker
+    include ActiveModel::Attributes
+    include ActiveModel::Model
```

Comment:
Maybe we don't need `ActiveModel::Conversion`, or `#initialize` and `persisted?` from `ActiveSupport::Model`, so maybe we could only use what the modules we need.